src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -12,51 +12,34 @@ import React from 'react';
 import asyncComponent from '../../Utilities/skeletonAsyncCard';
 import { injectIntl } from 'react-intl';
 import messages from '../../Messages';
+import { TimeStamp } from './../TimeStamp/TimeStamp';
 
 const ComplianceCard = asyncComponent(() => import('../../SmartComponents/Compliance/ComplianceCard'));
 const VulnerabilityCard = asyncComponent(() => import('../../SmartComponents/Vulnerability/VulnerabilityCard'));
+const SystemInventoryCard = asyncComponent(() => import('../../SmartComponents/SystemInventory/SystemInventoryCard'));
```

Comment:
something i was made aware of recently... we can use React.lazy and <Suspense /> for these imports for an example check it here https://github.com/RedHatInsights/insights-advisor-frontend/blob/prod-beta/src/SmartComponents/Overview/Dashboard.js#L30